NDTensors/ext/examples/NDTensorCUDA.jl

```diff
@@ -61,20 +61,17 @@ function main()
   #Currently this code fails with CUDA.allowscalar(false)
   # Because of outer calling the _gemm! function which calls a 
   # generic implementation
-  @allowscalar grad = gradient(f, cA, cB, cC, cD)
-  @allowscalar @test NDTensors.cpu(cB * cC * cD) ≈ NDTensors.cpu(grad[1])
+  grad = gradient(f, cA, cB, cC, cD)
+  @allowscalar @test cpu(cB * cC * cD) ≈ cpu(grad[1])
   @allowscalar @test (cB * cC * cD) ≈ grad[1]
   # Create a tuple of indices
-  decomp = (
-    dim(NDTensors.ind(grad[1], 1)),
-    dim(NDTensors.ind(grad[1], 2)) * dim(NDTensors.ind(grad[1], 3)),
-  )
+  decomp = (dim(ind(grad[1], 1)), dim(ind(grad[1], 2)) * dim(ind(grad[1], 3)))
   # Reshape the CuVector of data into a matrix
-  cuTensor_data = CUDA.reshape(NDTensors.data(storage(grad[1])), decomp)
+  cuTensor_data = CUDA.reshape(array(grad[1]), decomp)
```

Comment:
I still see a few instances of `CUDA.reshape` instead of just `reshape`.

Reply:
I saw this and wasn't sure why I used it over Base.reshape (i.e. reshape) but didn't look into it. I removed `CUDA.` in all instances now. Thanks